esbuild.config.js

```diff
@@ -30,6 +30,8 @@ const external = [
   '@lydell/node-pty-linux-x64',
   '@lydell/node-pty-win32-arm64',
   '@lydell/node-pty-win32-x64',
+  'tree-sitter',
```

Comment:
Fwiw @NTaylorMullen has asked us to not add tree sitter due to added external dependencies on at least 2 previous occasions. its worth validating the use case here before proceeding.